lib/ansible/executor/task_executor.py

```diff
@@ -605,6 +605,7 @@ def _get_connection(self, variables, templar):
                 try:
                     cmd = subprocess.Popen(['ssh','-o','ControlPersist'], stdout=subprocess.PIPE, stderr=subprocess.PIPE)
                     (out, err) = cmd.communicate()
+                    err = to_str(err)
```

Comment:
Odd -- on python3, to_str should just be an alias for to_unicode so there shouldn't be any difference.  On python2, they'd be different but it should work because of automatic coercion of byte strings to unicode strings in python2.